docs/design/usage-based-scheduling.md

```diff
@@ -57,9 +62,9 @@ tiers:
       - name: nodeorder
       - name: binpack
 metrics:                               # metrics server related configuration
-  type: prometheus                     # Optional, The metrics source type, prometheus by default, support prometheus and elasticsearch

```

Comment:
"prometheus_adapt" should be "prometheus_adaptor"

Reply:
done